pkg/bootkube/create.go

```diff
@@ -44,6 +46,9 @@ func CreateAssets(manifestDir string, timeout time.Duration) error {
 	if err := wait.Poll(5*time.Second, timeout, createFn); err != nil {
 		return fmt.Errorf("Failed to create assets: %v", err)
 	}
+
+	fmt.Println("Spawning self-hosted control plane...")
```

Comment:
Maybe just a verbiage change then? "Self-hosted cluster assets submitted"

We could also log the exact components we are creating here:
https://github.com/pbx0/bootkube/blob/c12e788f71e778a85314ea4601a4a9d42912e399/pkg/bootkube/create.go#L118

Then we also might want to expose any reason we might have exited here (if non-nil):
https://github.com/pbx0/bootkube/blob/c12e788f71e778a85314ea4601a4a9d42912e399/pkg/bootkube/bootkube.go#L112